src/utils/colors.ts

```diff
@@ -1,101 +1,60 @@
-/* tslint:disable */
-import color from 'onecolor'
+import { TinyColor as Color } from '@ctrl/tinycolor'
 import gray from 'gray-percentage'
 
 interface ColorMap {
   [colorName: string]: string
 }
 
-interface Colors {
-  accent: string
+interface GrayColorMap extends ColorMap {
+  calm: string
+}
+
+export interface ColorsTheme {
   bgDark: string
   defaultHover: string
   defaultLink: string
-  gray: ColorMap
-  primary: ColorMap
+  gray: GrayColorMap
+  primary: string
   secondary: ColorMap
-  ui: ColorMap
+  textDark: string
+  textDarkMuted: string
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)